tasks/dummy-dockerfile/dummy-dockerfile.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: inject-dockerfile
+spec:
+  description: >-
+    Tekton task to create a new dockerfile and inject it into a workspace
+  workspaces: 
+    - name: dockerfilews
+      description: The workspace where the dockerfile is stored
+      mountPath: /tmp
+  steps:
+    - name: create-dockerfile  
+      script: |
+       #!/usr/bin/env sh
+       set -xe
+       if [ "$(workspaces.dockerfilews.bound)" == "true" ] ; then
```

Comment:
I like having this `if` here to check if the workspace is bound. But then we should fail the task if nothing happens, right?